docs/using.rst

```diff
@@ -188,6 +190,24 @@ expected output is::
 .. _README: ../components/centos/centos-k8s-singlenode-setup/README.rst
 .. _Vagrantfile: ../components/centos/centos-k8s-singlenode-setup/Vagrantfile
 
+Vagrant 2 way folder sync
+=========================
+For basic usage please refer `Vagrant documentation. <https://www.virtualbox.org/manual/ch04.html#sharedfolders>`_
+
+In some scenarios 2 way bi-directional sync makes more sense as it is the simplest way to move files (e.g. code) between 
+the workstation and the Vagrant guest. Here are options for bi-directional sync you can use with ADB Vagrant box.
+
+* `vagrant-sshfs <https://github.com/dustymabe/vagrant-sshfs>`_ : It works with Linux/GNU, OS X and Microsoft Windows.
```

Comment:
Wouldn't you need examples on how to use these things? Also, I believe NFS and SMB are not even tested, are they?